project/scripts/bootstrapCmdTests

```diff
@@ -83,3 +83,12 @@ clear_out "$OUT"
 ./bin/scalac -d "$OUT/out.jar" tests/pos/i12973.scala
 echo "Bug12973().check" | TERM=dumb ./bin/scala -cp "$OUT/out.jar" > "$tmp" 2>&1
 grep -qe "Bug12973 is fixed" "$tmp"
+
+echo "testing -sourcepath with inlining"
+cwd=$(pwd)
+java_prop="-Dpack.version.file=$cwd/dist/target/pack/VERSION"
+sbt_test_command=";clean;prepareSources;compile;copyChanges;compile"
```

Comment:
isn't the property used here to do something equivalent to:
```suggestion
sbt_test_command="++$(cat dist/target/pack/VERSION)!;clean;prepareSources;compile;copyChanges;compile"
```

Reply:
I would like to do that but VERSION is a file with three properties inside

Reply:
found a solution without using java properties